eng/common/TestResources/New-TestResources.ps1

```diff
@@ -119,6 +119,22 @@ function MergeHashes([hashtable] $source, [psvariable] $dest) {
     }
 }
 
+function BuildBicepFile([System.IO.FileSystemInfo] $file) {
+    $tmp = $env:TEMP ? $env:TEMP : [System.IO.Path]::GetTempPath()
+    $prefix = ($file | Resolve-Path -Relative) -replace '\.|/', '_'
+    $templateFilePath = Join-Path $tmp ($prefix + '.test-resources.compiled.json')
```

Comment:
Not sure how much I care right now but we might need to figure out a clean-up strategy.

Reply:
In the devops environment, the directory the value I set TEMP to will get cleaned up after each build. For local testing, I would assume that the user is responsible for it (either cleaning up their own temp directory, or if they override it, managing that). I'm not sure how much else we can do without introducing too many side effects (e.g. if you're running the script from multiple repos, how do you know which script owns which files?). I guess we could follow the linux convention of dot directories in home? It's another case where I'm hesitant to introduce too much complexity.

Reply:
Could we delete this generated file after we deploy the resource at the end of script?